stack/dashboard/base/builder.sh

```diff
@@ -96,7 +96,7 @@ sed -i "s'\$DIR/config'/etc/wazuh-dashboard'g" ./bin/opensearch-dashboards-plugi
 sed -i 's/NODE_OPTIONS="$OSD_NODE_OPTS_PREFIX $OSD_NODE_OPTS $NODE_OPTIONS"/NODE_OPTIONS="$OSD_NODE_OPTS_PREFIX $OSD_NODE_OPTS $NODE_OPTIONS"\n/g' ./bin/use_node
 sed -i 's/exec "${NODE}"/NODE_ENV=production exec "${NODE}" ${NODE_OPTIONS} /g' ./bin/use_node
 # Replace the redirection to `home` in the header logo
-sed -i "s'/app/home'/app/wazuh'g" ./src/core/target/public/core.entry.js
+sed -i "s'/app/home'/app/wz-home'g" ./src/core/target/public/core.entry.js
```

Comment:
This change affects other components such as the DevOps repositories or the installation assistant itself, but I don't see that it has been taken into account since the assistant test code continues with the old path

An issue could be opened to the team responsible for each case to adapt it to the development (post merge) if it has not been created (I don't see it)

Reply:
Issue: https://github.com/wazuh/internal-devel-requests/issues/394